server/src/cli.rs

```diff
@@ -119,8 +119,28 @@ pub struct Cli {
     pub trino_auth: Option<String>,
     pub trino_schema: Option<String>,
     pub trino_catalog: Option<String>,
+
+    // audit log vars
+    pub audit_log_target: Option<String>,
+    pub audit_log_target_username: Option<String>,
+    pub audit_log_target_password: Option<String>,
+    pub audit_log_target_tls_verify: bool,
+    pub audit_log_target_headers: HashMap<String, String>,
```

Comment:
This could just as well be an [`http::HeaderMap`](https://docs.rs/http/latest/http/header/struct.HeaderMap.html)